apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/LoadedFilterManager.java

```diff
@@ -115,8 +138,8 @@ public void setEntityManager(EntityManager entityManager) {
 
   /** Called to finish initialization of the manager */
   @PostConstruct
-  public void init() {
-    refreshFilters();
+  public synchronized void init() {
+    transactionTime = fetchLastLoadedBatchCreated();
```

Comment:
Here's the algorithm change. `fetchLastLoadedBatchCreated` will have a consistent duration because of the index on `LoadedBatches`